docker/pool/.dockerignore

```diff
@@ -0,0 +1,5 @@
+build-screen/node_modules*
+build-screen/bower_components*
+coverage*
+**.swp
```

Comment:
Sorry, I thought this syntax works well in `.dockerignore` like in `.gitignore`.
Then I remove these statements  because it's necessary to add `.dockerignore` because these file is small and doesn't affect upload-context size very much.